rn/components/ManageTeam.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { View, Button, TextInput, Alert } from "react-native";
+import styles from "../stylesheet";
+import { Text, ListItem } from "react-native-elements";
+
+import { useAuth } from "../providers/AuthProvider";
+
+export function ManageTeam({}) {
+  const { user } = useAuth();
+  const [newTeamMember, setNewTeamMember] = useState(null);
+  const [teamMemberList, setTeamMemberList] = useState([]);
+
+  // getTeam calls the backend function getMyTeamMembers to retrieve the
+  // team members of the logged in user's project
+  const getTeam = async () => {
+    try {
+      const teamMembers = await user.functions.getMyTeamMembers([]);
+      setTeamMemberList(teamMembers);
+    } catch (err) {
+      Alert.alert("An error occurred while getting team members", err);
+      throw `an error occurred while getting team members: ${err}`;
+    }
+  };
+  getTeam();
+
+  // addTeamMember calls the backend function addTeamMember to add a
+  // team member to the logged in user's project
+  const addTeamMember = async () => {
+    try {
+      await user.functions.addTeamMember(newTeamMember);
+      getTeam();
+    } catch (err) {
+      Alert.alert("An error occurred while adding a team member", err.message);
+      throw `an error occurred while adding a team member: ${JSON.stringify(
+        null,
+        err,
+        2
+      )}`;
+    }
+  };
+
+  // removeTeamMember calls the backend function removeTeamMember to remove a
+  // team member from the logged in user's project
+  const removeTeamMember = async (email) => {
+    try {
+      const removeTeamMemberResult = await user.functions.removeTeamMember(
+        email
+      );
```

Comment:
Should we use removeTeamMemberResult or at least log it?

Reply:
Fixed - removed that variable